packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -121,12 +122,17 @@ export default defineComponent({
     }
   },
   computed: {
+    ...mapGetters(['user']),
+
     showNoResults() {
       return this.searchResults.every(({ result }) => !result.values.length)
     },
     availableProviders() {
       return this.providerStore.availableProviders
     },
+    isSearchBarEnabled() {
+      return this.availableProviders.length && this.user?.id
```

Comment:
Could you please add a setup function with
```js
return {
  isUserContext: useUserContext({ store })
}
```
and use `this.isUserContext` instead of `this.user?.id`?